lib/react_on_rails/helper.rb

```diff
@@ -324,8 +324,10 @@ def load_pack_for_component(component_name)
 
```

Comment:
we should rename to `load_pack_for_generated_component`

changes to support non-generated don't make sense b/c you can have many components defined in one bundle if doing by hand.